editor/src/clj/editor/model.clj

```diff
@@ -55,62 +60,79 @@
         [])
       (:animation-ids animation-set-info))))
 
-(g/defnk produce-pb-msg [name mesh material textures skeleton animations default-animation]
+(g/defnk produce-pb-msg [name mesh materials skeleton animations default-animation]
   (cond-> {:mesh (resource/resource->proj-path mesh)
-           :material (resource/resource->proj-path material)
-           :textures (mapv resource/resource->proj-path textures)
+           :materials (mapv
+                        (fn [material]
+                          (-> material
+                              (update :material resource/resource->proj-path)
+                              (update :textures
+                                      (fn [textures]
+                                        (mapv #(update % :texture resource/proj-path) textures)))))
```

Comment:
No, it shouldn't: when we produce materials (see `output materials`), we clean them by removing all texture bindings without an assigned texture (IIRC, the engine will crash otherwise). Because of that, it is expected that texture bindings in this code block are never `nil`.